src/tracking-buffer/writable-tracking-buffer.js

```diff
@@ -1,27 +1,40 @@
+// @flow
+
 const bigint = require('./bigint');
 
 const SHIFT_LEFT_32 = (1 << 16) * (1 << 16);
 const SHIFT_RIGHT_32 = 1 / SHIFT_LEFT_32;
 const UNKNOWN_PLP_LEN = new Buffer([0xfe, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff]);
 const ZERO_LENGTH_BUFFER = new Buffer(0);
 
+export type Encoding = 'utf8' | 'ucs2' | 'ascii';
```

Comment:
I _think_ these are all the encodings that we use together with `WritableTrackingBuffer`. If we find more later one, we can add them here.